.travis.yml

```diff
@@ -12,20 +12,17 @@ cache:
     - "node_modules"
 before_install:
   - "npm -g install npm@latest"
+  - "curl -Lo travis_after_all.py https://raw.githubusercontent.com/dmakhno/travis_after_all/master/travis_after_all.py"
 before_script:
   - "npm run lint"
 script:
   - "npm test"
-after_success:
+after_success:  # travis_after_all.py is needed due to travis-ci/travis-ci#1548 & travis-ci/travis-ci#929
   - "npm run coveralls"
+  - "python travis_after_all.py"
+  - "export $(cat .to_export_back)"
+  - "npm run semantic-release || true"
```

Comment:
Semrel is using https://github.com/semantic-release/condition-travis to check whether it is running on master